src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -440,7 +425,7 @@ bool BaseCouplingScheme::addComputedTime(
   _computedTimeWindowPart += timeToAdd;
 
   // Check validness
-  bool valid = math::greaterEquals(getNextTimeStepMaxSize(), 0.0, _minTimeStepSize);
+  bool valid = math::greaterEquals(getNextTimeStepMaxSize(), 0.0, math::NUMERICAL_ZERO_DIFFERENCE);
```

Comment:
```suggestion
  bool valid = math::greaterEquals(getNextTimeStepMaxSize(), 0.0);
```
should be the default, right?